### ISSUE TITLE
Bookmarks pane does not scroll correctly with touch/touchpad

### ISSUE THREAD

**Issue by EpicWolverine** [sumatrapdfreader/sumatrapdf#480]
When using a touchscreen or touchpad to scroll/pan the main PDF area, it moves perfectly with one-to-one tracking and momentum when flick scrolling. However, the bookmarks pane only scrolls one entry at a time at a fixed speed when scrolled in this manner. This is fine when done with a scroll wheel, but doesn't make sense with a touchscreen or two-finger touchpad where the expected behavior is similar to the behavior in the main PDF area.

**Comment by Clear-up** [comment]
Totally the same problem on my Surface 3!

**Comment by GitHubRulesOK** [comment]
@kjk 
Since the necessary behaviour is to step from one line to the next I think the current behaviour is on that basis both fast (on my underpowered touchpad system) and as smooth as can be expected from "stepping" I suggest if you agree that it is working per design then this issue can be closed ?

**Comment by kjk** [comment]
I can reproduce this on my touch screen but I'm not sure if we can improve it.

Scroll of main area is handled by our code.

Bookmarks are displayed using standard tree control. Not sure if it's even possible to take over scrolling.

I couldn't find any references to slow scrolling with touch on the web.

Maybe can be tweaked with `TreeView_SetAutoScrollInfo` but docs for this are non-existent and I can't find any code that uses it to see what good values could be.

**Comment by kjk** [comment]
Maybe try http://www.danielmoth.com/Blog/tvsexautohscroll.aspx

**Comment by kjk** [comment]
I tried:
```
        SendMessageW(hwnd, TVM_SETEXTENDEDSTYLE, TVS_EX_AUTOHSCROLL, TVS_EX_AUTOHSCROLL);
        WPARAM pixelsPerSec = 80;
        LPARAM redrawInterval = 10;
        SendMessageW(hwnd, TVM_SETAUTOSCROLLINFO, pixelsPerSec, redrawInterval);
```
and it didn't help.

I compared with scrolling in explorer's treeview and we are on par so closing this as a fix, even if possible, would probably require some unreasonable level of messing with tree view internals.

BTW: for me only touch scrolling is slow. Scrolling with touchpad or trackpad is fast.

It could also be some system-level config value or value specific to a given touch driver.
